apps/prairielearn/package.json

```diff
@@ -122,6 +122,7 @@
     "loopbench": "^2.0.0",
     "lru-cache": "^10.2.2",
     "mathjax": "^3.2.2",
+    "mdast": "^3.0.0",
```

Comment:
It looks like `mdast` is a deprecated old copy of `remark`? See https://www.npmjs.com/package/mdast

Reply:
Removed in https://github.com/PrairieLearn/PrairieLearn/pull/10089/commits/e221bc7181b2c232012702ba49c48150738bae79